kubernetes-ts-guestbook/README.md

```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new?template=https://github.com/pulumi/examples/tree/master/kubernetes-ts-guestbook/components)
 
-# Kubernetes Guestbook (Two Ways)
+# Simple and Component-based Kubernetes Guestbook Apps
```

Comment:
Again, why "simple"?

Reply:
In the actual example, it mentions showing the Guestbook "two ways", and calls the variants these:
```
simple/ is a straight port of the original YAML.
components demonstrates benefits of using a real language, namely eliminating boilerplate through the use of real component abstractions.
```

Reply:
Open to your alternative suggestion but leaving it as is for now